modules/rkhunter/manifests/monitoring.pp

```diff
@@ -44,11 +44,19 @@
   }
 
   # Script to run rkhunter as a passive check
-  file { '/etc/cron.daily/rkhunter-passive-check':
+  file { '/usr/local/bin/rkhunter-passive-check':
     ensure  => present,
     mode    => '0755',
     content => template('rkhunter/rkhunter-passive-check.erb'),
   }
+
+  # Run it twice a day to stop superfluous alerting
+  cron::crondotdee { 'rkhunter-passive-check':
+    command => '/usr/local/bin/rkhunter-passive-check',
+    hour    => '6, 14',
```

Comment:
Does a space between the two hour specifiers do what you'd expect? I didn't think you could have whitespace.